### ISSUE TITLE
Code block in list item breaks numbered list

### ISSUE THREAD

**Issue by jessetan** [sphinx-contrib/sphinxcontrib-confluencebuilder#175]
When a numbered list item contains a code block, the generated Confluence wiki markup contains empty lines around the `{code}` block. Confluence interprets this as the end of the list, instead of a code block within the list item. The next list item will start a new list (restarting the numbering), instead of continuing the list.

RST input:
```rst
#.  step 1

    .. code-block:: none

        code block in step 1

#.  step 2
    more text on the same line
#.  step 3

    line 2 of step 3

#.  step 4
```

Generated output:
```confluence
# step 1

{code:linenumbers=false|language=none}
code block in step 1
{code}

# step 2 more text on the same line
# step 3
line 2 of step 3
# step 4
```
Gets rendered by Confluence as:
![screen shot 2019-02-21 at 15 28 03](https://user-images.githubusercontent.com/3284111/53175962-646e9200-35ed-11e9-817b-f7dd4489c519.png)

Expected output:
```
# step 1
{code:linenumbers=false|language=none}
code block in step 1
{code}
# step 2 more text on the same line
# step 3
line 2 of step 3
# step 4
```

Gets rendered by Confluence as:
![screen shot 2019-02-21 at 15 28 13](https://user-images.githubusercontent.com/3284111/53175986-6fc1bd80-35ed-11e9-9a11-6f629da069ea.png)

Found in version 0.9.

**Comment by jdknight** [comment]
@jessetan, thanks for the report. This (and many other formatting issues) have been resolved in the switch to storage format #121. The changes are available on `master` and should be made available via PyPI in an upcoming v1.0 release (see #171).

An example output of your sample to Confluence Cloud:
![image](https://user-images.githubusercontent.com/1834509/53288841-b33a3a00-375b-11e9-8d3b-93bf9c473458.png)

If you want, you can use the [most recent sources](https://sphinxcontrib-confluencebuilder.readthedocs.io/en/latest/install.html#master) by using the following command in your environment:

    pip install git+https://github.com/tonybaloney/sphinxcontrib-confluencebuilder

Ideally, the v1.0 release should be available within the next month.

**Comment by jdknight** [comment]
Version 1.0 of this extension has been released. It is assumed this issue primarily covers an issue which is available in the most stable version. If another issue is attempted to be outlined but has been overlooked, please feel free to open a new issue with more details. Marking this issue as closed.
